src/app/components/dynamic-table/dynamic-table.component.ts

```diff
@@ -27,7 +32,7 @@ export class DynamicTableComponent
     /* eslint-disable  @typescript-eslint/no-explicit-any */
     @Input() public tableColumns?: any[];
     /* eslint-disable  @typescript-eslint/no-explicit-any */
-    @Input() public tableSource: any[];
+    @Input() public tableSource: (SearchOverviewDatasetsInterface[] | SearchHistoryInterface[] | DataSchemaField[] | undefined);
```

Comment:
Can we think of using generics here?
https://www.typescriptlang.org/docs/handbook/2/generics.html#generic-classes

Reply:
fixed